tests/conftest.py

```diff
@@ -0,0 +1,4 @@
+# let pytest see our fixtures & configuators
+from taichi.testing import get_conftest
+
+get_conftest(globals())
```

Comment:
What does `conftest` mean?

Reply:
Confront test, I guess, to be run before any test is ran, since pytest will automatically run `tests/conftest.py` as a setup script.